apps/web/cypress/tests/notifications-editor.spec.ts

```diff
@@ -538,7 +583,7 @@ describe('Notifications Creator', function () {
       });
     });
 
-    it.only('should be able to delete a step', function () {
```

Comment:
because of this `.only`, other tests were not running... I've removed that `.only` and run all of them locally, they were passing so I've decided to leave that...

Reply:
Oh great catch! @novuhq/novu-team check it out aswell, we should try to pay a really close attention to this.

Reply:
@LetItRock created #947 to make sure this won't happen in the future too